src/viewer/overlay/DeckList.tsx

```diff
@@ -8,12 +8,18 @@ import { CardsProps, sort as cardSorting, withCards } from "../../utils/cards";
 import { DecklistPosition } from "../../utils/config";
 import { getCopiableDeck } from "../../utils/hearthstone";
 import { withProps } from "../../utils/styled";
-import { TwitchExtProps, withTwitchExt } from "../../utils/twitch";
+import {
+	TwitchExtConsumer,
+	TwitchExtConsumerArgs,
+	TwitchExtProps,
+	withTwitchExt,
+} from "../../utils/twitch";
 import CardTile from "../CardTile";
 import { CopyDeckIcon, HSReplayNetIcon, PinIcon, UnpinIcon } from "../icons";
 
 interface PositionProps {
 	position: DecklistPosition;
+	usingNewExtComponentDesign: boolean;
```

Comment:
Let's flip this naming logic so it mirrors the URL param and because soon it will be default:

```suggestion
	legacyComponentDesign?: boolean;
```